### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by rijkvanzanten**

### CODE REVIEW

File: week2/app.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        </li>
+      `;
+    },
+
+    /**
+     * Create single list item
+     * @param {Object} asteroid Asteroid meta info object
+     * @returns {String} Html element
+     */
+    listItem(asteroid) {
```
Comment (IanCStewart) [inline week2/app.js:137]: @rijkvanzanten & @joostf This doesn't seem semantically correct. Listitem which is an article that gets nested in a `<li>`.... this kinda smells 💩

File: week2/style.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
+html {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+    color: white;
+}
+
+body {
+    min-height: 100vh;
```
Comment (IanCStewart) [inline week2/style.css:7]: Use 100% and add height: 100% to html. This will break on ios safari

File: week2/style.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    transform: scale(.98);
+}
+
+body > img:first-of-type[data-loading=false] {
+    opacity: .3;
+    transform: scale(1);
+}
+
+body > h1, body > p {
+    color: white;
```
Comment (IanCStewart) [inline week2/style.css]: Couldn't this be inherit? Since color on body is also white

File: week2/style.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    font-style: italic;
+    max-width: 35em;
+}
+
+h2 {
+    margin: 0;
+}
+
+.asteroid-list {
+    color: white;
```
Comment (IanCStewart) [inline week2/style.css]: See previous comment

File: week2/style.css [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+    width: 35rem;
```
Comment (IanCStewart) [inline week2/style.css]: Are u sure u want to set this width this hard?

File: week2/style.css [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+}
+
+.asteroid-list article {
+    border-bottom: 1px solid white;
+    transition: all 250ms cubic-bezier(0.4, 0.0, 0.2, 1);
+    position: relative;
+}
+
+.asteroid-list article[data-hazardous=false] {
+    color: gray;
```
Comment (IanCStewart) [inline week2/style.css]: please use the real version `grey` 🇨🇦

File: week2/style.css (hunk 2 above)
Comment (rijkvanzanten) [reply week2/style.css:7]: The app won't work at all on iOS 🔥👍🏻

File: week2/app.js (hunk 1 above)
Comment (rijkvanzanten) [reply week2/app.js:137]: Output example:

```html
<ol>
  <li>
    <time></time>
    <article></article>
    <article></article>
    <article></article>
  </li>
  <li>
    <time></time>
    <article></article>
    <article></article>
    <article></article>
  </li>
</ol>
```

**Pull request merged**